### DISCUSSION TITLE
Named tuple / array with literal strings

### DISCUSSION THREAD

**Discussion by o-az** [open-circle/valibot#349] [Q&A] [answered]
I want to validate against the following array: `['ens', 'primary-list', 'following', 'followers']`
Is this the right way to do it?
```ts
const schema = v.tuple([v.literal('ens'), v.literal('primary-list'), v.literal('following'), v.literal('followers')])
```

also what is the equivalent of this https://zod.dev/?id=default `.default` in valibot?

**Comment by fabian-hiller** [comment, marked as answer]
Yes, this looks good to me. With Valibot you use [`optional`](https://valibot.dev/api/optional/) with a default argument as the second parameter.
